packages/shared/src/components/auth/OnboardingLogs.tsx

```diff
@@ -139,9 +139,9 @@ const EXPERIENCE_TO_SENIORITY: Record<
   NOT_ENGINEER: 'not_engineer',
 };
 
-export const trackAnalyticsSignUp = ({
+export const trackLogsSignUp = ({
```

Comment:
Maybe we can lose `track` at all here if we fear it triggers any review. `LogSignUp` might be just enough